pkg/apis/storage/validation/validation.go

```diff
@@ -230,22 +229,6 @@ func ValidateVolumeAttachmentUpdate(new, old *storage.VolumeAttachment) field.Er
 
 var supportedVolumeBindingModes = sets.NewString(string(storage.VolumeBindingImmediate), string(storage.VolumeBindingWaitForFirstConsumer))
 
-// validateVolumeBindingMode tests that VolumeBindingMode specifies valid values.
-func validateVolumeBindingMode(mode *storage.VolumeBindingMode, fldPath *field.Path) field.ErrorList {
-	allErrs := field.ErrorList{}
-	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeScheduling) {
-		if mode == nil {
-			allErrs = append(allErrs, field.Required(fldPath, ""))
```

Comment:
also remove the feature gate from the defaulting behavior in `SetDefaults_StorageClass` so that nil always defaults to VolumeBindingImmediate